app/models/status.rb

```diff
@@ -266,7 +266,7 @@ def non_sensitive_with_media?
   end
 
   def reported?
-    @reported ||= Report.where(target_account: account).unresolved.exists?(['? = ANY(status_ids)', id])
+    @reported ||= Report.where(target_account: account).unresolved.exists?(['? = ANY(status_ids)', id]) || AccountWarning.where(target_account: account).exists?(['? = ANY(status_ids)', id.to_s])
```

Comment:
Line could be slightly compressed with `account.targeted_reports...` in first part and `account.strikes...` in second part (I think those are same query?)

Possible future idea ... I think there are a few more models with the "collection of `status_ids` as array column" setup in place ... could pull out some shared querying ability or scope or something for those. Might be contingent on them all being exact same column type, which I haven't double checked.

Reply:
They are currently not all the same exact column type, unfortunately. `AccountWarning` have them be strings for some reason :/